webpack.config.js

```diff
@@ -51,6 +51,10 @@ module.exports = {
         test: /.(eot|svg|ttf|woff|woff2)$/,
         use: [{ loader: 'file-loader?name=font/[name]__[hash:base64:5].[ext]' }],
       },
+      {
+        test: /.(eot|svg|ttf|woff|woff2)$/,
+        use: [{ loader: 'file-loader?name=font/[name]__[hash:base64:5].[ext]' }]
+      }
     ],
```

Comment:
Is this a mistake? Seems like duplicates

Reply:
lol probably a mistake, yeah - I'm scared of config files!

Reply:
Yeah 54-57 should be removed

Reply:
Done, thx!